src/Umbraco.Web/Dashboards/ContentDashboard.cs

```diff
@@ -9,19 +12,30 @@ public class ContentDashboard : IDashboard
     {
         public string Alias => "contentIntro";
 
-        public string[] Sections => new [] { "content" };
+        public string[] Sections => new[] { "content" };
 
         public string View => "views/dashboard/default/startupdashboardintro.html";
 
         public IAccessRule[] AccessRules
         {
             get
             {
-                var rules = new IAccessRule[]
+                IAccessRule[] rules;
+                var dashboardConfig = Path.Combine(IOHelper.MapPath(SystemDirectories.Config), "content.dashboard.access.config.js");
+
+                if (File.Exists(dashboardConfig))
+                {
+                    var rawJson = File.ReadAllText(dashboardConfig);
+                    rules = JsonConvert.DeserializeObject<AccessRule[]>(rawJson);
```

Comment:
Maybe we want to create a backing field for the rules? If I attach a debugger to this get property, I can see that it's called three times every time the content page is loaded, which means that we'll be reading a file from disk three times each time the content section is loaded which seems inefficient.

The major downside to this though is that the site has to rebooted before the changes take effect, so it might not work.

Reply:
Good find @nikolajlauridsen, will refactor!